thirdeye/thirdeye-frontend/app/pods/manage/alerts/performance/template.hbs

```diff
@@ -0,0 +1,121 @@
+            <a class="te-anomaly-table__cell-link" {{action "toggleSortDirection" "resrate"}}>
+              Response rate
+              <i class="te-anomaly-table__glyph te-anomaly-table__glyph--small glyphicon {{if sortColumnResrateUp "glyphicon-menu-up" "glyphicon-menu-down"}}"></i>
+            </a>
+          </th>
+           <th class="te-anomaly-table__cell-head">
+            <a class="te-anomaly-table__cell-link" {{action "toggleSortDirection" "precision"}}>
+              Precision
+              <i class="te-anomaly-table__glyph te-anomaly-table__glyph--small glyphicon {{if sortColumnPrecisionUp "glyphicon-menu-up" "glyphicon-menu-down"}}"></i>
+            </a>
+          </th>
+        </tr>
+      </thead>
+
+      <tbody>
+        {{#each applications as |app|}}
+          <tr class="te-anomaly-table__row">
+            <td class="te-anomaly-table__cell te-anomaly-table__list--stronger">{{app.name}}</td>
+            <td class="te-anomaly-table__cell">{{app.alerts}}</td>
+            <td class="te-anomaly-table__cell">
+              {{performance-tooltip
+                data=app.data.totalAlerts
+                category="Anomalies"
+                viewTotals=viewTotals
+              }}
+            </td>
+            <td class="te-anomaly-table__cell">
+              {{performance-tooltip
+                data=app.data.userReportAnomaly
+                category="User reported anomalies"
+                viewTotals=viewTotals
+              }}
+            </td>
+            <td class="te-anomaly-table__cell">
+              <div class="performance-metric">
+                {{performance-tooltip
+                 data=app.data.totalResponses
+                 category="Total Responses"
+                 viewTotals=viewTotals
+                }}
+              </div>
+              <div class="performance-metric__subset">
+                ( True: {{performance-tooltip
+                       data=app.data.trueAnomalies
+                       category="True anomalies"
+                       viewTotals=viewTotals
+                     }} |
+                False: {{performance-tooltip
+                     data=app.data.falseAlarm
+                     category="False anomalies"
+                     viewTotals=viewTotals
+                   }} |
+                New: {{performance-tooltip
+                     data=app.data.newTrend
+                     category="New trend anomalies"
+                     viewTotals=viewTotals
+                   }} )
+            </div>
+            </td>
+            <td class="te-anomaly-table__cell">
+              {{app.data.responseRate}}
+              <span class="te-anomaly-table__list--smaller">%</span>
+            </td>
+            <td class="te-anomaly-table__cell">
+              {{app.data.precision}}
+              <span class="te-anomaly-table__list--smaller">%</span>
+            </td>
+          </tr>
+        {{/each}}
+      </tbody>
+    </table>
```

Comment:
I might be missing something here, but is there a reason we can't utilize `ember-models-table` here?

Reply:
Would be a good case for it - I just wanted to use a raw table in the first demo to keep it simple and get feedback quickly ... we don't need filtering, for example, just sorting for now.

Reply:
In that case, can you add a `TODO` to refactor this and replace it with `ember-models-table`, so that we don't have to maintain multiple types of tables?